### DISCUSSION TITLE
A successful Octolamp

### DISCUSSION THREAD

**Discussion by ziti** [martinwoodward/octolamp#15] [Show and tell]
I have an Ender 3 v2 Neo, which only has a 200mm x 200mm print bed so in Cura, I scaled it to 90% of original size and it worked quite well.

Edit: as pointed out to me below, I had the print bed size settings in Cura incorrect. The Ender 3 v2 Neo has a 220mm x 220mm sized bed.

![IMG_8414](https://user-images.githubusercontent.com/152570/219823356-ed94f15f-e8bc-488c-b225-2eb57e3e3b22.jpeg)

**Comment by vanZeben** [comment]
I also have a Ender 3 v2 Neo, wanted to point out that the bed size is actually 220x220mm and you can print this in full (I've printed one myself). Only caveat is you _might_ have to wipe the initial filament purge line either from your gcode, or just clean it up on the fly so that the actual print doesn't overlap. I don't know that it does actually overlap but it looked close enough that I just cleaned up the purge line first.

**Reply by ziti** [reply]
Wow, you're right! The printer settings in Cura that I downloaded somewhere had the bed 200x200. Thank you so much for pointing that out!
